services/app-web/src/pages/SubmissionSummary/SubmissionSummary.test.tsx

```diff
@@ -187,6 +189,46 @@ describe('SubmissionSummary', () => {
             ).toBeNull()
         })
 
+        it('extracts the correct dates from the submission and displays them in tables', async () => {
```

Comment:
please add this test to the SubmissionRevisionSummary.test.tsx too cuz now we need to test both places

Reply:
done